src/components/ZeroWidthView/index.native.js

```diff
@@ -0,0 +1,5 @@
+function ZeroWidthView() {
```

Comment:
last note - please add comments why we return null for native - just to not forget the reason @ayazalavi